webpack_loader/templatetags/webpack_loader.py

```diff
@@ -12,10 +13,12 @@ def render_bundle(bundle_name):
     bundle = get_bundle(bundle_name)
     tags = []
     for chunk in bundle:
-
-        if chunk['name'].endswith('.js'):
-            url = chunk.get('publicPath') or chunk['url']
-            tags.append('<script type="text/javascript" src="{}"></script>'.format(url))
-        elif chunk['name'].endwith('.css'):
-            tags.append('<link type="text/css" href="{}" rel="stylesheet">'.format(url))
+        path = chunk.get('path', None)
+        url = staticfiles_storage.url(path)
+        name = str(chunk.get('name'))
+        if url:
+            if name.endswith('.js'):
+                tags.append('<script type="text/javascript" src="{}"></script>'.format(url))
+            elif name.endswith('.css'):
+                tags.append('<link type="text/css" href="{}" rel="stylesheet">'.format(url))
     return '\n'.join(tags)
```

Comment:
@destos I tested it and everything seems to work. BTW why did we change the URL generation code?